components/lib/autocomplete/AutoComplete.js

```diff
@@ -430,7 +430,7 @@ export const AutoComplete = React.memo(
 
         useUpdateEffect(() => {
             if (searchingState) {
-                ObjectUtils.isNotEmpty(props.suggestions) ? show() : hide();
+                ObjectUtils.isNotEmpty(props.suggestions) ? show() : props.showEmptyMessage ? show() : hide();
```

Comment:
You could do `ObjectUtils.isNotEmpty(props.suggestions) || props.ShowEmptyMessage ? show() : hide();`